### PR TITLE
Start & stop commands

### PR DESCRIPTION
**Pull request by sroze**
This PR contains two things:
- It changes the name of the `up` command to `start`. It looks like to have some confusion about what the `up` command is doing, and as it's actually just starting the containers, IMO it should be named like that.
- Introduce a `stop` command that stop the running containers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,8 @@ The following commands are available:
 
 ### Project commands
 
-- [`up` command](docs/cmd-up.md) starts the Docker environment for the project.
+- [`start` command](docs/cmd-start.md) starts the Docker environment for the project.
```
Comment (ciaranmcnulty) [inline README.md:46]: :+1: this is clearer.

File: docs/cmd-stop.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# `stop` command
+
+This stop running containers of a docker-compose-based project.
```
Comment (ciaranmcnulty) [inline docs/cmd-stop.md]: 'This stops'

File: src/Compose/ComposeExecutableFinder.php [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        $this->executableFinder = new ExecutableFinder();
+    }
+
+    /**
+     * @return string
+     */
+    public function find()
+    {
+        if (null === ($executable = $this->executableFinder->find('docker-compose'))) {
+            throw new \RuntimeException('Unable to find docker-compose binary');
```
Comment (ciaranmcnulty) [inline src/Compose/ComposeExecutableFinder.php]: Maybe suggest `docker:install` here?

Comment (sroze) [inline src/Compose/ComposeExecutableFinder.php]: Good point !

File: src/Cli/StopCommand.php [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->userInteraction->writeTitle('Stopping application containers');
+
+        pcntl_exec($this->composeExecutableFinder->find(), ['stop']);
```
Comment (ciaranmcnulty) [inline src/Cli/StopCommand.php]: I feel like the instances of `pcntl_exec` should be handled by a replacable object (especially for testing).

In this example the command doing direct Shell access makes me think it's a problem.

Comment (sroze) [inline src/Cli/StopCommand.php]: @ciaranmcnulty updated, they now are ran by the process runner.

**Comment by tonypiper** [comment]
LGTM (have run it locally).

**Pull request merged**